@types/pal/image.d.ts

```diff
@@ -0,0 +1,30 @@
+declare module 'pal/image' {
+    type ImageSource = import('pal/image/types').ImageSource;
+    type IMemoryImageSource = import('pal/image/types').IMemoryImageSource;
```

Comment:
```suggestion
    export type ImageSource = import('pal/image/types').ImageSource;
    export type IMemoryImageSource = import('pal/image/types').IMemoryImageSource;
```

Reply:
Here I was thinking of using pal/image or using relative paths to reference.

Reply:
It's better to use pal/image, after engine modulized, we prefer to define type in `.d.ts` files